guides/common/modules/con_building_cloud_images.adoc

```diff
@@ -5,9 +5,6 @@ Use this section to build and register images to {ProjectName}.
 
 You can use a preconfigured {RHEL} KVM guest QCOW2 image:
 
-* https://access.redhat.com/downloads/content/69/ver=/rhel---7/[Latest RHEL 7 KVM Guest Image]
```

Comment:
All changes in this file need to be reverted.

Reply:
Addressed :+1: